source/_integrations/sensibo.markdown

```diff
@@ -85,6 +85,17 @@ For Pure devices, these sensors are available:
 - PM2.5
 - Pure Boost Sensitivity
 
+## Custom Services
+
+### Pure Boost
+
+You can configure your Pure Boost settings by using the services `Enable Pure Boost` and `Disable Pure Boost` that is provided.
+
```

Comment:
It would be nice to tell them the service name `sensibo.enable_pure_boost` and `sensibo.disable_pure_boost` so they don't have to hunt for it